lib/perl/Genome/Model/Tools/DetectVariants2/SpeedseqSv.t.d/svs.hq.sv.NA12878.20slice.30X.aligned.bam.readdepth.bed

```diff
@@ -0,0 +1,2 @@
+20_slice	51001	53900	0.559811
+20_slice	260701	263200	0.504321
```

Comment:
It'd be good to make sure there's a newline at the end of these BED files.  Some of our tools don't work well when it's missing.